tests/window/minimising.rs

```diff
@@ -6,11 +6,13 @@ fn main() {
     // TODO: Combine this with `resizing` once multiple_windows is simpler than
     // it is currently.
     App::new()
-        .insert_resource(WindowDescriptor {
-            title: "Minimising".into(),
-            ..Default::default()
-        })
-        .add_plugins(DefaultPlugins)
+        .add_plugins(DefaultPlugins.build().set(WindowPlugin {
```

Comment:
`.build()` is not needed here on `DefaultPlugins` correct?

Reply:
correct!